types/alloy/tslint.json

```diff
@@ -0,0 +1,13 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
Please do not disable lint rules in new definitions.

Reply:
Got it :+1: